examples/astro/public/style/home.css

```diff
@@ -0,0 +1,52 @@
+:root {
+    --font-mono: Consolas, "Andale Mono WT", "Andale Mono", "Lucida Console", "Lucida Sans Typewriter", "DejaVu Sans Mono", "Bitstream Vera Sans Mono", "Liberation Mono", "Nimbus Mono L", Monaco, "Courier New", Courier, monospace;
+    --color-light: #F3F4F6;
+}
+
+@media (prefers-color-scheme: dark) {
+    :root {
+        --color-light: #1F2937;
+    }
+}
+
+a {
+    color: inherit;
+}
+
+header > div {
+    font-size: clamp(2rem, -0.4742rem + 6.1856vw, 2.75rem);
+}
+
+header > div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+header h1 {
+    font-size: 1em;
+    font-weight: 500;
+}
+header img {
+    width: 2em;
+    height: 2.667em;
+}
+
+h2 {
+    font-weight: 500;
```

Comment:
The formatting on this file looks off. We probably need to add Prettier to this repo, but in the meantime, could you run Prettier over these files?